src/org/thoughtcrime/securesms/util/views/ProgressDialog.java

```diff
@@ -51,7 +51,7 @@ private void setupProgressBar() {
         if (progressBar != null) {
             progressBar.getIndeterminateDrawable()
                     .setColorFilter(ContextCompat.getColor(getContext(), R.color.delta_accent), PorterDuff.Mode.SRC_IN);
-            progressBar.setIndeterminate(indeterminate);
+            this.indeterminate = indeterminate;
```

Comment:
i do not understand why progressBar.setIndeterminate() is superfluous. imu, this swaps drawables, starts animations etc. it is just a system call that tells the used object how it shall appear.

but even if the call is not needed, `this.indeterminate = indeterminate` is a void statement, why does this avoid some smell?

but, tbh, i did not dived into details :)